### PR TITLE
#824 Add ability to disable fade on <Alert />

### PR DESCRIPTION
**Pull request by electriccode**

### CODE REVIEW

File: src/UncontrolledAlert.js [hunk 1]
```diff
@@ -1,12 +1,22 @@
...
+  fade: true
+};
+
+class UncontrolledAlert extends Component {
   constructor(props) {
     super(props);
 
     this.state = { isOpen: true };
     this.toggle = this.toggle.bind(this);
+    this.fade = this.props.fade;
```
Comment (TheSharpieOne) [inline src/UncontrolledAlert.js]: I don't think any changes are needed to this file, but at the very least, this change does nothing.

**Comment by electriccode** [comment]
@TheSharpieOne done

**Comment by TheSharpieOne** [comment]
Yeah, we usually just don't define the props for the uncontrolled components but when I was cleaning it up I thought it would be interesting if we exposed the original props through. I probably should have only done it for prop types and not default props but I didn't give it much thought.

**Comment by electriccode** [comment]
I can add back `prop types` from `Alert` component, but this is not observed at other places so lets stick with this for now.

**Pull request merged**